posts/_markdown/2023_hierarqcal.md

```diff
@@ -0,0 +1,91 @@
+
+![](../images/2023_hierarqcal_motifs.drawio.svg){.center}
+
+The motifs on the first level we call primitives, the bottom-to-top QCNN has two: a convolution of stride 1 and pooling from bottom to top. One level up the hierarchy we create another motif that alternates between the two primitives, a convolution-pooling unit. The third level motif is three convolution-pooling units repeated. This final motif, which is one directed graph, contains the full QCNN architecture. You can check the paper ([Lourens et al.](https://arxiv.org/abs/2210.15073#)) for the full theoretical framework, but basically, the effect of a primitive is based on its hyperparameters and the effect of its predecessor. This way, their individual and combined architectural effects are captured, enabling  them to be dynamically stacked one after another to form second level $l=2$ motifs. Stacking these stacks in different ways constitutes higher-level motifs until a final level $l=L$, where one motif constitutes the entire QCNN architecture. 
+
+This hierarchical view is particularly useful for architecture search, algorithms that automatically design model architectures (such as for the QCNN). The main idea is to use smaller operations as building blocks for larger ones which result in an expressive representation that mimics design elements typically performed by humans. In classical machine learning literature, the field of neural architecture search (NAS) explores architecture representations as part of automated neural network construction. In particular, hierarqcal was inspired by the hierarchical representation presented in this paper by [Liu et al.](https://openreview.net/pdf?id=BJQRKzbA-).
+
+<img class="center"src="../images/2023_hierarqcal_dalle_img.png" style="width:200px;border: 1px black solid; margin-right: 10px;" align="left"/>
+
+Speaking of inspiration, our logo comes from [Dall E 2](https://openai.com/dall-e-2/) presented with the following prompt: *A robot building itself with artificial intelligence, pencil drawing*
+
+Which touches on the idea of a robot building another, in some way hierarqcal is the pencil that allows you or a robot to "draw" quantum circuits.
+<br/>
+
+There's a bunch of [docs and tutorials](https://matt-lourens.github.io/hierarqcal/getting_started.html#basic-usage) that can get you started with building QCNNs or creating algorithms that generate them. Some useful things to mention are:
+
+- The package is quantum computing framework agnostic, so you can use it with any framework (Qiskit, Cirq, Pennylane) or your own, as long as it's in python. This is because the architecture representation is just a data structure and the package allows you to interact with it.
```

Comment:
```suggestion
- The package is quantum computing framework agnostic, so you can use it with any framework (e.g., Qiskit, Cirq, Pennylane) or your own, as long as it's in python. This is because the architecture representation is just a data structure and the package allows you to interact with it.
```